pkg/services/servicetest/run_test.go

```diff
@@ -0,0 +1,99 @@
+package servicetest
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+)
+
+func TestRunHealthy(t *testing.T) {
+	for _, test := range []struct {
+		name    string
+		s       services.Service
+		expFail bool
```

Comment:
I tried to validate error messages too, but they contain whole stacks from testify internals.